samtranslator/model/preferences/deployment_preference_collection.py

```diff
@@ -102,7 +102,7 @@ def _codedeploy_iam_role(self):
             ],
         }
         iam_role.ManagedPolicyArns = [
-            ArnGenerator.generate_aws_managed_policy_arn("service-role/AWSCodeDeployRoleForLambda")
+            ArnGenerator.generate_aws_managed_policy_arn("service-role/AWSCodeDeployRoleForLambdaLimited")
```

Comment:
there is some conflict in this part, we already applied this change as following:
https://github.com/aws/serverless-application-model/blob/579353c793b9c779c0f869f7bb7e4a16b16f8767/samtranslator/model/preferences/deployment_preference_collection.py#L112-L121
my understanding is Policy AWSCodeDeployRoleForLambda will be deprecated only in MVP regions

Reply:
Thank you for reviewing this,

No, we will be deprecating AWSCodeDeployRoleForLambda policy in all regions.